server/lib/nl/common/constants.py

```diff
@@ -761,6 +763,8 @@
         "Relative to the Average Yearly Min Temperature between 1980-2010, the Predicted Min Temperature difference with 95% chance, at least once in the year, according to a CMIP6 Ensemble model as per SSP 245 (intermediate) scenario.",
     "DiffRelativeToAvg_1980_2010_MinTemp_Daily_Hist_95PctProb_LessThan_Atleast1DayAYear_CMIP6_Ensemble_SSP585":
         "Relative to the Average Yearly Min Temperature between 1980-2010, the Predicted Min Temperature difference with 95% chance, at least once in the year, according to a CMIP6 Ensemble model as per SSP 585 (pessimistic) scenario.",
+    "dc/e9gftzl2hm8h9":
+        "Total time spent on a single commute across all workers",
```

Comment:
in a week?

Reply:
It looks like this is for a single commute actually. The original question here:https://www.census.gov/acs/www/about/why-we-ask-each-question/commuting/ suggests a single commute, and dividing the variable value by "Population: Outside of Home" also gives the average commute time cited by Census ACS on their website, which appears to be for a single commute.

Reply:
Ah, got it.  From [1] "Total time spent on a one-way commute from home to work across all workers."

[1] - https://www2.census.gov/programs-surveys/acs/tech_docs/subject_definitions/2021_ACSSubjectDefinitions.pdf